.github/workflows/test-e2e.yaml

```diff
@@ -113,7 +113,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: zulu
-          java-version: 8
+          java-version: 11
```

Comment:
> - make it possible to build Maestro on all Java 11 and higher. Compilation is faster with each release, and [has other advantages](https://jakewharton.com/build-on-latest-java-test-through-lowest-java)
> - keep minimum Java required to run Maestro at 8. Note: for building, JDK11 will be the new minimum.

Following your comment, this test-e2e/test should work with java-version: 8 right? Otherwise, this means that maestro is unable to run at 8.

Reply:
> this test-e2e/test should work with java-version: 8 right?

This is correct. (This is actually one of the things I wanted to accomplish with the `test-e2e.yaml` - to always verify we work on Java 8)

Now, this change (in `test-cloud` job) is actually a mistake, it should've been 8.

On Android (the `test-local` job) it's currently not 8, because none of the "setup Android & start emulator" GitHub Actions support installing an older SDK Command-Line Tools release (newer require either Java 11 or 17). [I'm hoping this will change soon](https://github.com/Malinskiy/action-android/issues/86).

Reply:
So that means we can only test if `maestro cloud` works on java 8? Actually using maestro locally it's still TBD?

Reply:
> Actually using maestro locally it's still TBD? 

It already works as well, but the workaround I used is to compile on Java 8, and then run on Java 11 (because Android tools require at least Java 11). But now we will not be able to compile anymore on Java 8, so we have to find a way run on Java 8.

> Perhaps we could use devices from Maestro Cloud and connect them remotely using adb and therefore use the maestro cli?

We could, but this sounds a bit complex (but I'd love to discuss more if you think it makes sense here). For now though I'd prefer to go with the simplest possible solution - find a way to use old Android Command-Line Tools+Java 8 and run emulator on GitHub Actions.